simpl/server.py

```diff
@@ -169,3 +267,59 @@ def run(self, handler):
         tornado.ioloop.IOLoop.instance().start()
 
 bottle.server_names['xtornado'] = XTornadoServer
+
+
+def attach_parser(subparser):
+    """Given a subparser, build and return the server parser."""
+    return subparser.add_parser(
+        'server',
+        help='Run a bottle based server',
+        parents=[
+            CONFIG.build_parser(
+                add_help=False,
+                # might need conflict_handler
+            ),
+        ],
+    )
+
+
+def run(conf):
+    """Simpl server command line interface."""
+    if isinstance(conf.adapter_options, list):
+        options = {key: val for _dict in conf.adapter_options
+                   for key, val in _dict.items()}
+    elif conf.adapter_options is None:
+        options = {}
+    else:
+        options = copy.copy(conf.adapter_options)
+
+    # waiting for https://github.com/bottlepy/bottle/pull/783
+    if conf.app and (os.getcwd() not in sys.path):
+        sys.path.append(os.getcwd())
+
+    try:
+        if conf.reloader and not os.getenv('BOTTLE_CHILD'):
+            LOG.info("Running bottle server with reloader.")
+        return bottle.run(
+            app=conf.app,
+            server=conf.server,
+            host=conf.host,
+            port=conf.port,
+            interval=conf.interval,
+            reloader=conf.reloader,
+            quiet=conf.quiet_server,
+            debug=conf.debug_server,
+            **options
+        )
+    except KeyboardInterrupt:
```

Comment:
I think the bottle file watcher (for reloading) issues KeyboardInterrupt to stop the process that is running the old set of files. Myabe you need to check if you're running in the server or watcher process (see https://github.com/checkmate/simpl/pull/82/files#diff-7f2f24942936abfb2a1b215c243bd28bR301).